libfwupdplugin/fu-common.c

```diff
@@ -3282,6 +3286,44 @@ fu_common_align_up (gsize value, guint8 alignment)
 	return value_new;
 }
 
+/**
+ * fu_common_kenv:
```

Comment:
How about fu_common_bsd_kenv to make it clearer to future selfs this is only intended for use with bsd?

Reply:
is kenv used on macos too? that's the bit i wasn't sure of.

Reply:
Yes, this seems used on OS-X too. Maybe a better approach would be to have a libfwupdplugin/fu-kenv.[c|h] with a symbol like fu_kenv_get_string()? That might be better for the long term.

Reply:
sounds good

Reply:
Done.